src/Orleans.BroadcastChannel/Hosting/ChannelHostingExtensions.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Orleans.BroadcastChannel;
+using Orleans.BroadcastChannel.SubscriberTable;
+using Orleans.Configuration;
+using Orleans.Hosting;
+using Orleans.Runtime;
+
+namespace Orleans.Hosting
+{
+    public static class ChannelHostingExtensions
+    {
+        public static ISiloBuilder AddBroadcastChannel(this ISiloBuilder @this, string name, Action<BroadcastChannelOptions> configureOptions)
```

Comment:
Perhaps this should be plural, `AddBroadcastChannels`, like ASP.NET's `AddControllers` extension method: https://github.com/dotnet/aspnetcore/blob/3ea008c80d5cc63de7f90ddfd6823b7b006251ff/src/Mvc/Mvc/src/MvcServiceCollectionExtensions.cs#L89

Although there is a difference: this adds support for `BroadcastChannel`, whereas `AddControllers` adds the individual controllers (and `AddMvc` adds support for them)